src/app/components/content-page/content-page.component.ts

```diff
@@ -20,8 +20,9 @@ export class ContentPageComponent implements OnInit {
   faPlusSquare = faPlusSquare;
   faSearch = faSearch;
   isSearchCollapsed = true;
-
   user: User;
+  faHome = faHome;
+  faUser = faUser;
   firebaseStorage: any;
   images: Image[];
 
```

Comment:
So faHome and faUser should be moved to faSearch and faPlusSquare.
Please add the open line back since this makes the code more readable in the sense that the user will know at the top we initialise the icons and underneath it we initialise other variables, which can be grouped later on.

Reply:
I did this ! I don't think the **user** will read the code though.

Reply:
:)